src/pkg/packager/lint/lint_test.go

```diff
@@ -9,136 +9,249 @@ import (
 	"errors"
 	"fmt"
 	"os"
-	"path/filepath"
 	"testing"
 
-	"github.com/defenseunicorns/zarf/src/config"
-	"github.com/defenseunicorns/zarf/src/pkg/packager/composer"
+	"github.com/defenseunicorns/zarf/src/pkg/variables"
 	"github.com/defenseunicorns/zarf/src/types"
 	goyaml "github.com/goccy/go-yaml"
 	"github.com/stretchr/testify/require"
 )
 
-const badZarfPackage = `
-kind: ZarfInitConfig
-metadata:
-  name: init
-  description: Testing bad yaml
-
-components:
-- name: first-test-component
-  import:
-    not-path: packages/distros/k3s
-- name: import-test
-  import:
-    path: 123123
-`
-
-const goodZarfPackage = `
-x-name: &name good-zarf-package
-
-kind: ZarfPackageConfig
-metadata:
-  name: *name
-  x-description: Testing good yaml with yaml extension
-
-components:
-  - name: baseline
-    required: true
-    x-foo: bar
-
-`
-
 func readAndUnmarshalYaml[T interface{}](t *testing.T, yamlString string) T {
 	t.Helper()
 	var unmarshalledYaml T
 	err := goyaml.Unmarshal([]byte(yamlString), &unmarshalledYaml)
-	if err != nil {
-		t.Errorf("error unmarshalling yaml: %v", err)
-	}
+	require.NoError(t, err)
 	return unmarshalledYaml
 }
 
-func TestValidateSchema(t *testing.T) {
+func TestZarfSchema(t *testing.T) {
+	t.Parallel()
 	getZarfSchema := func(t *testing.T) []byte {
 		t.Helper()
 		file, err := os.ReadFile("../../../../zarf.schema.json")
-		if err != nil {
-			t.Errorf("error reading file: %v", err)
-		}
+		require.NoError(t, err)
 		return file
 	}
```

Comment:
Let's read the schema file once at the start of this test function and reuse the schema byte slice. This way, we avoid reading the file from disk multiple times.

Reply:
Yup, good catches